ARM/STM32/svd/stm32f40x/stm32_svd-adc.ads

```diff
@@ -718,45 +719,45 @@ package STM32_SVD.ADC is
    --  Analog-to-digital converter
    type ADC1_Peripheral is record
       --  status register
-      SR    : SR_Register;
+      SR    : aliased SR_Register;
```

Comment:
It's the wavefront branch of SVD2Ada, waiting for the next GPL to become the master branch.

The rationale is that sometimes you need to pass register around. Having aliased values here helps.